src/backend/distributed/utils/resource_lock.c

```diff
@@ -1104,22 +1114,16 @@ AcquireDistributedLockOnRelations(List *relationIdList, LOCKMODE lockMode)
 	 */
 	bool startedLockCommand = false;
 
+	/* Used to terminate partial lock commands */
```

Comment:
minor: we can move this declaration inside of the if statement where it is used.